yt_dlp/extractor/lsm.py

```diff
@@ -272,7 +283,8 @@ def _real_extract(self, url):
             '_type': 'url_transparent',
             'id': video_id,
             **traverse_obj(data, {
-                'url': ('playback', 'service', 'url', {url_or_none}),
+                # Fall back to hls_url for audio streams
+                'url': ('playback', 'service', ('url', 'hls_url'), {url_or_none}),
```

Comment:
If there is no `url` but only `hls_url`, we need to call `_extract_m3u8_formats` and return `formats` in the info dict instead of `url`